code/modules/mob/living/carbon/xenomorph/Facehuggers.dm

```diff
@@ -343,30 +350,32 @@
 	stat = CONSCIOUS
 	jump_timer = addtimer(CALLBACK(src, PROC_REF(try_jump)), time_between_jumps, TIMER_OVERRIDE|TIMER_STOPPABLE|TIMER_UNIQUE)
 
-/obj/item/clothing/mask/facehugger/proc/go_idle() //Idle state does not count toward the death timer.
+/obj/item/clothing/mask/facehugger/proc/go_idle(delete_timer = TRUE) //Idle state does not count toward the death timer.
 	if(stat == DEAD || stat == UNCONSCIOUS)
 		return
 
 	stat = UNCONSCIOUS
 	icon_state = "[initial(icon_state)]_inactive"
-	if(jump_timer)
-		deltimer(jump_timer)
-	jump_timer = null
+	if(delete_timer)
+		if(jump_timer)
+			deltimer(jump_timer)
+		jump_timer = null
```

Comment:
To be idle, we should not have a jump timer OR try jump should do nothing if we are in an idle state.

The current problem is that now the hugger is dying while idle when the only behavior we want to change is for them to expire in hands (and ideally not to have a way to prolong).

So my recommendation is to revert the changes to this proc and to restore the CONSCIOUS check in try_jump. The stat being UNCONSCIOUS is basically your indicator for whether we are idle or not.